doc/src/manual/faq.md

```diff
@@ -29,7 +29,7 @@ you can redefine types and constants.  You can't import the type names into `Mai
 to be able to redefine them there, but you can use the module name to resolve the scope.  In other
 words, while developing you might use a workflow something like this:
 
-```
+```julia
```

Comment:
guess that would be equivalent to treating docstrings as a julia specific markdown extension, which i guess we already do with even numbers of backticks using math mode. worth an issue at documenter for discussion maybe.